examples/pxScene2d/src/shell.js

```diff
@@ -74,8 +74,6 @@ if (false)
 	  var code  = e.keyCode;
     var flags = e.flags;
 
-    console.log("SHELL: onPreKeyDown:", code, " key: ", keys.name(code), ", ", flags);
```

Comment:
For the time being, let's leave the key prints in place as they are useful for debugging.

Reply:
Do we have a way to conditionally enable this?  They might be useful for debugging for developers of pxscene, but not so much for people developing applications on pxscene.

Reply:
Let's compromise and check the environment variable to know whether or not to log these messages:
- Use process.env.RT_LOG_LEVEL: Valid values are "debug", "info" , "warn" , "error", "fatal".  Only log if value is debug or info.
- Please add a consistent prefix for these messages while you're here; something like "SHELL:"; that will help people to grep these out when they wish. 
- Long term we should look at adding a logging API wrapper so we don't need a check for log level sprinkled throughout.